static/app/components/onboarding/gettingStartedDoc/step.tsx

```diff
@@ -8,12 +8,14 @@ import {space} from 'sentry/styles/space';
 
 export enum StepType {
   INSTALL = 'install',
+  REGISTER = 'register',
```

Comment:
worth checking usages of this type in case code replied on all values to render some logic instead `Object.keys(StepType)`